src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -1239,6 +1239,43 @@ def test_sql_db_long_term_retention(
             checks=[NoneCheck()])
 
 
+@record_only()
+class SqlServerDbGeoRestoreScenarioTest(ScenarioTest):
+    def test_sql_db_geo_restore(
```

Comment:
```suggestion
class SqlServerDbGeoRestoreScenarioTest(ScenarioTest):
    @record_only()
    # using fixed resources because of long time preperation for geo-redundant backup
    # need to change resources for others who want to rerecord this test
    def test_sql_db_geo_restore(
```